fixtures/pages-functions-wasm-app/wasm/add.wat

```diff
@@ -0,0 +1,7 @@
+(module
```

Comment:
`add.wasm` is generated from this file via `wat2wasm add.wat -o addwasm `. Technically this file is not needed, but wanted to keep it for readability purposes